### DISCUSSION TITLE
delete

### DISCUSSION THREAD

**Discussion by xtreh** [uNetworking/uWebSockets.js#384] [Q&A] [answered]
...

**Comment by ghost** [comment]
This is an open ended question without enough info to make anything out of it. Your image doesn't say anything other than there's a gap of 12 seconds at the very browser (?). This doesn't really rule out anything other than there's not, not a gap of 12 seconds.

**Comment by xtreh** [comment]
....

**Comment by ghost** [comment, marked as answer]
This is not really how I want to do things - you should debug on your side until you have concrete, absolute info on this being a bug. Otherwise I spend ages and ages doing everybody's homework.

**Comment by ghost** [comment]
Closing until you have an actual concrete bug report.
